### PR TITLE
DOCS-8625-WSC-configs-edits-kt

### PR DESCRIPTION
**Pull request by kevintroller**
WSC 1.6.1 Additional Configuration doc.

### CODE REVIEW

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc [hunk 1]
```diff
@@ -0,0 +1,92 @@
+= Web Service Consumer Additional Configuration Information - Mule 4
```
Comment (kbostrom) [inline web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc]: I see topics called "Additional Configuration" and I always wonder what that means: what are these things, optional? Why aren't they with the basic config topic?
Consider something like this, based on the actual content here:
```suggestion
= Customize Transport and Secure Web Service Consumer Connector - Mule 4
```

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+:page-aliases: connectors::web-service/web-service-consumer-configure.adoc
+
+Set additional configuration for Anypoint Connector for Web Service Consumer (Web Service Consumer Connector), such as custom HTTP Transport and Web Service Security both in Anypoint Studio and in the XML editor.
```
Comment (kbostrom) [inline web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc]: ditto, something like:
```suggestion
Add custom HTTP transport and web service security to Anypoint Connector for Web Service Consumer (Web Service Consumer Connector), both in Anypoint Studio and in the XML editor.
```
Could be made more robust by briefly stating WHY.

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+== Configure Custom HTTP Transport
+
+By default, the connector uses a simple nonprotected HTTP configuration to dispatch all outgoing SOAP messages. For cases where a more advanced HTTP configuration is required, you can parameterize the Web Service Consumer Connector with an HTTP Connector configuration to dispatch the messages.
```
Comment (kbostrom) [inline web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc]:
```suggestion
By default, the connector uses a simple unprotected HTTP configuration to dispatch all outgoing SOAP messages. When you need more advanced HTTP transport, parameterize Web Service Consumer Connector with an HTTP Connector configuration to dispatch the messages.
```
Could we specify under what circumstances our audience might need this? What does "more advanced" mean, specifically? What aspects are improved or advanced?

If this is an issue of security, then perhaps this topic set should be titled "Secure Web Service Consumer Connector - Mule 4" and subtitled "Secure Message Transport" and "Secure Message Content"?

Comment (kbostrom) [reply web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc]: Also, is there such an action as "parameterizing with a configuration"?
You might want to check with a peer writer to ensure that this is the best way to word this.

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+In the following example, you declare and reference an HTTP Requester configuration with HTTP Basic authentication in the *Transport Configuration* parameter of the Web Service Consumer Connector configuration, so the connector dispatches each requests using the advanced configuration:
```
Comment (kbostrom) [inline web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc]:
```suggestion
The following example illustrates how you might declare and reference an HTTP Requester configuration with HTTP Basic authentication in the *Transport Configuration* parameter of the Web Service Consumer Connector configuration to ensure that the connector dispatches each request using the advanced configuration:
```
I'm a little confused by "Basic authentication" ensuring "using the advanced configuration"?
Also...which advanced configuration? Do we mean just "an" advanced configuration? Again, what is the improvement or advancement being configured?

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+. In Studio, open the *Global Element Properties* window of the Web Service Consumer Connector.
+. On the *Transport* tab, for *Transport Configuration*, select `HTTP Transport Configuration`. +
+The *Requester Config* field automatically populates `HTTP_Request_configuration`.
+. Click *OK*
+
+In Studio, configuring Custom HTTP Transport looks like this:
```
Comment (kbostrom) [inline web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc]:
```suggestion
In Studio, configuring custom HTTP transport looks like this:
```

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+.Configure Custom HTTP Transport
+image::web-service-consumer-configure-transport.png[Configure Custom HTTP Transport]
+
+In the *Configuration XML* tab, the reference of the HTTP Requester configuration `<http:request-config name="basic-auth">` is under the Web Service Consumer configuration `<wsc:custom-transport-configuration>`, and looks as follows:
```
Comment (kbostrom) [inline web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc]:
```suggestion
In the *Configuration XML* tab, the reference of the HTTP Requester configuration `<http:request-config name="basic-auth">` is under the Web Service Consumer configuration `<wsc:custom-transport-configuration>`:
```

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+        <wsc:custom-transport-configuration>
+            <wsc:http-transport-configuration requesterConfig="basic-auth"/>
+        </wsc:custom-transport-configuration>
+    </wsc:connection>
+</wsc:config>
+----
+
+
+
+== Configure Web Service Security
```
Comment (kbostrom) [inline web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc]: See previous comments about topic set and subtopic titles.

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+
+The WS-Security protocol specifies how you can enforce integrity and confidentiality for messages and enables the communication of various security token formats. Its main focus is the use of XML Signature and XML Encryption to provide end-to-end security.
```
Comment (kbostrom) [inline web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc]:
```suggestion
The WS-Security protocol enables you to use various security token formats to enforce message integrity and confidentiality. This protocol uses XML Signature and XML Encryption to provide end-to-end security.
```

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+
+WS-Security describes how to:
```
Comment (kbostrom) [inline web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc]:
```suggestion
WS-Security describes how to:
```

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc [hunk 10]
```diff
@@ -0,0 +1,92 @@
...
+
+* Sign SOAP messages to assure integrity. Signed messages also provide nonrepudiation.
```
Comment (kbostrom) [inline web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc]:
```suggestion
* Sign SOAP messages to ensure integrity and provide nonrepudiation
```

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc [hunk 11]
```diff
@@ -0,0 +1,92 @@
...
+* Encrypt SOAP messages to assure confidentiality.
```
Comment (kbostrom) [inline web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc]:
```suggestion
* Encrypt SOAP messages to ensure confidentiality
```

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc [hunk 12]
```diff
@@ -0,0 +1,92 @@
...
+* Attach security tokens to ascertain the sender's identity.
```
Comment (kbostrom) [inline web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc]:
```suggestion
* Attach security tokens to ascertain the sender's identity
```

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc [hunk 13]
```diff
@@ -0,0 +1,92 @@
...
+
+The Web Service Consumer Connector supports the following mechanisms:
```
Comment (kbostrom) [inline web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc]:
```suggestion
Web Service Consumer Connector supports the following mechanisms:
```

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc (hunk 7 above)
Comment (kbostrom) [reply web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc]: I looked up WS-Security and found "Web Services Security is an extension to SOAP to apply security to Web services."
So, the NAME is WS-Security or Web Services Security, but the thing provided is security for web services, or web service security. 
Please GLOBALLY search for and verify that the capitalization used is appropriate in each context.

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc [hunk 14]
```diff
@@ -0,0 +1,92 @@
...
+
+* Timestamp
+* Signature or signature verification
+* Username
+* Encryption or decryption
+
+In the following example, you configure a username token security strategy as Web Service Security for the connector:
```
Comment (kbostrom) [inline web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc]:
```suggestion
The following example illustrates how you might configure a username token security strategy as web service security for the connector:
```

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc [hunk 15]
```diff
@@ -0,0 +1,92 @@
...
+
+. In Studio, open the *Global Element Properties* window of the Web Service Consumer Connector.
+. On the *Security* tab, for *Username token security strategy*, select `Edit inline`.
+. Set the *Username*, and *Password* fields, and set *Add nonce* and *Add created* fields to `true`.
```
Comment (kbostrom) [inline web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc]:
```suggestion
. Complete the *Username* and *Password* fields, and set *Add nonce* and *Add created* fields to `true`.
```

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc [hunk 16]
```diff
@@ -0,0 +1,92 @@
...
+. Click *OK*.
+
+In Studio, configuring Web Service Security looks as follows:
```
Comment (kbostrom) [inline web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc]:
```suggestion
In Studio, configuring web service security looks like this:
```

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc [hunk 17]
```diff
@@ -0,0 +1,92 @@
...
+
+.Configure Web Service Security
+image::web-service-consumer-configure-wsecurity.png[Configure Web Service Security]
+
+In the *Configuration XML* tab, configuring Web Service Security looks as follows:
```
Comment (kbostrom) [inline web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc]:
```suggestion
In the *Configuration XML* tab, configuring web service security looks like this:
```

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc [hunk 18]
```diff
@@ -0,0 +1,92 @@
...
+<wsc:config name="with-custom-transport">
+    <wsc:connection wsdlLocation="http://www.host.com?WSDL" service="Service" port="Port">
+        <wsc:web-service-security>
+            <wsc:username-token-security-strategy username="admin" password="textPassword" addCreated="true" addNonce="true"/>
+        </wsc:web-service-security>
+    </wsc:connection>
+</wsc:config>
+----
+
+//After you complete <summary of config topics>, you can try the Examples.
```
Comment (kbostrom) [inline web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc]: Isn't this a line from the connector template that you need to customize and include here?

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc (hunk 2 above)
Comment (kbostrom) [reply web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc]: Are the referenced web services actually Amazon Web Services (AWS)?
I'm a little confused about whether this topic set refers to the common terms "web service" or "web services" or some proper noun such as Amazon Web Services or Web Service Consumer Connector.
Please review and ensure proper presentation throughout.

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc (hunk 6 above)
Comment (kbostrom) [reply web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc]: ...or should this be "...is under the web service consumer configuration?

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc (hunk 7 above)
Comment (kbostrom) [reply web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc]: P.S. We no longer init cap "web" when it is not part of a proper noun.

File: web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc [hunk 19]
```diff
@@ -0,0 +1,92 @@
...
+
+== See Also
+
+* https://help.mulesoft.com[MuleSoft Help Center]
+* xref:web-service-consumer-reference.adoc[Web Service Consumer Reference]
```
Comment (kbostrom) [inline web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-config-topics.adoc]: ?
```suggestion
* xref:web-service-consumer-reference.adoc[Web Service Consumer Connector Reference]
```

**Review by kbostrom — DISMISSED** [review]
Edited

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**